CSharp/BatchExplorer/ViewModels/MainViewModel.cs

```diff
@@ -1014,6 +1014,29 @@ public CommandBase OpenRDP
         }
 
         /// <summary>
+        /// Get SSH settngs for the selected ComputeNode
+        /// </summary>
+        public CommandBase GetSSH
+        {
+            get
+            {
+                return new CommandBase(
+                    (o) =>
+                    {
+                        var task = this.GetSSHSettingsAsync(this.SelectedComputeNode);
```

Comment:
Minor thing here, but may cause some confusion. When adding credentials to a linux node, a user can use ssh and/or a password... meansing ssh may not be there.